2996_reflection/reflection.md

```diff
@@ -4592,52 +4600,52 @@ using enum operators;
 
 <center>Table 1: Enum class `operators` [meta.reflection.operators]</center>
 
-|Constant|Corresponding operator|
-|:-|:-|
-|`op_new`|`operator new`|
-|`op_delete`|`operator delete`|
-|`op_array_new`|`operator new[]`|
-|`op_array_delete`|`operator delete[]`|
-|`op_co_await`|`operator co_await`|
-|`op_parentheses`|`operator()`|
-|`op_square_brackets`|`operator[]`|
-|`op_arrow`|`operator->`|
-|`op_arrow_asterisk`|`operator->*`|
-|`op_tilde`|`operator~`|
-|`op_exclamation_mark`|`operator!`|
-|`op_plus`|`operator+`|
-|`op_minus`|`operator-`|
-|`op_asterisk`|`operator*`|
-|`op_solidus`|`operator/`|
-|`op_percent`|`operator%`|
-|`op_caret`|`operator^`|
-|`op_ampersand`|`operator&`|
-|`op_pipe`|`operator|`|
-|`op_equals`|`operator=`|
-|`op_plus_equals`|`operator+=`|
-|`op_minus_equals`|`operator-=`|
-|`op_asterisk_equals`|`operator*=`|
-|`op_solidus_equals`|`operator/=`|
-|`op_percent_equals`|`operator%=`|
-|`op_caret_equals`|`operator^=`|
-|`op_ampersand_equals`|`operator&=`|
-|`op_pipe_equals`|`operator|=`|
-|`op_equals_equals`|`operator==`|
-|`op_exclamation_equals`|`operator!=`|
-|`op_less`|`operator<`|
-|`op_greater`|`operator>`|
-|`op_less_equals`|`operator<=`|
-|`op_greater_equals`|`operator>=`|
-|`op_three_way_compare`|`operator<=>`|
-|`op_ampersand_ampersand`|`operator&&`|
-|`op_pipe_pipe`|`operator||`|
-|`op_less_less`|`operator<<`|
-|`op_greater_greater`|`operator>>`|
-|`op_less_less_equals`|`operator<<=`|
-|`op_greater_greater_equals`|`operator>>=`|
-|`op_plus_plus`|`operator++`|
-|`op_minus_minus`|`operator--`|
-|`op_comma`|`operator,`|
+|Constant|Corresponding operator|Operator symbol name|
+|:-|:-|:-|
+|`op_new`|`operator new`|`new`|
```

Comment:
Should the right-most column be strings? So `"new"` instead of `new`, etc?

Reply:
Discussed offline; I didn't want to imply an encoding. It's more like an abstract character sequence.